linux/mcompile/keymap/deadkey.cpp

```diff
@@ -1,468 +1,284 @@
 #include "keymap.h"
 #include "deadkey.h"
 
-vec_dword_1D createLine(std::string  first, std::string second, KMX_DWORD number, std::string nameresult) {
-	vec_dword_1D line;
-	line.push_back(convertNamesTo_DWORD_Value(first));
-	line.push_back(convertNamesTo_DWORD_Value(second));
-	//line.push_back(convertNamesTo_DWORD_Value(nameresult));
-	line.push_back(number);
-	return line;
-}
-
-std::vector<DeadKey*> create_deadkeys_by_basechar() {
-	std::vector<DeadKey*> alDead;
-	vec_dword_2D dk_ComposeTable;
+std::vector<DeadKey *> create_deadkeys_by_basechar() {
+  std::vector<DeadKey *> alDead;
+  vec_dword_2D dk_ComposeTable;
 
   create_DKTable(dk_ComposeTable);
 
-	for( int i=0; i < (int) dk_ComposeTable.size()-1; i++) {
-		DeadKey *dk2 = new DeadKey(dk_ComposeTable[i][0]);
-		for ( int j=0; j< (int) dk_ComposeTable.size();j++) {
-			if(( dk_ComposeTable[i][0] == dk_ComposeTable[j][0]) && (IsKeymanUsedChar(dk_ComposeTable[j][1])))
-				dk2->KMX_AddDeadKeyRow(dk_ComposeTable[j][1],dk_ComposeTable[j][2]);
-		}
-		alDead.push_back(dk2);
-	}
-	return alDead;
+  for (int i = 0; i < (int)dk_ComposeTable.size() - 1; i++) {
+    DeadKey *dk2 = new DeadKey(dk_ComposeTable[i][0]);
+    for (int j = 0; j < (int)dk_ComposeTable.size(); j++) {
+      if ((dk_ComposeTable[i][0] == dk_ComposeTable[j][0]) && (IsKeymanUsedChar(dk_ComposeTable[j][1])))
+        dk2->KMX_AddDeadKeyRow(dk_ComposeTable[j][1], dk_ComposeTable[j][2]);
+    }
+    alDead.push_back(dk2);
+  }
+  return alDead;
 }
 
-void refine_alDead(KMX_WCHAR dk, std::vector<DeadKey*> &dkVec, std::vector<DeadKey*> *p_All_Vec) {
-	if (dk == 0)
-		return;
-
-	for (int j=0; j < (int) (*p_All_Vec).size()-1;j++) {
-		if (dk == (*p_All_Vec)[j]->KMX_GetDeadCharacter()) {
-			if(! found_dk_inVector(dk, dkVec)) {
-				dkVec.push_back((*p_All_Vec)[j]);
-				return;
-			}
-			else return;
-		}
-	}
+void refine_alDead(KMX_WCHAR dk, std::vector<DeadKey *> &dkVec, std::vector<DeadKey *> &r_All_Vec) {
+  if (dk == 0)
+    return;
+
+  for (int j = 0; j < (int)r_All_Vec.size() - 1; j++) {
+    if (dk == (r_All_Vec)[j]->KMX_GetDeadCharacter()) {
+      if (!found_dk_inVector(dk, dkVec)) {
+        dkVec.push_back((r_All_Vec)[j]);
+        return;
+      } else
+        return;
```

Comment:
This could be simplified as:

```suggestion
        dkVec.push_back(r_All_Vec[j]);
      }
      return;
```